src/infohud.asm

```diff
@@ -436,6 +440,35 @@ ih_update_hud_code:
     RTL
```

Comment:
A few lines up from here, there is existing code that prints a decimal.  Just saying you don't need a new graphic.

Reply:
Fair enough. I really just needed something to put there, and wanted it to be something useful instead of blank/garbage tiles. I'll swap to the old decimal for these in case the new one gets replaced later.